sandbox/src/LargePayload.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+export default function LargePayload() {
+
+  const makePayload = size => () => {
+    const payload = new Uint8Array(size * 1024);
+    window.alloy("event", {
+      beacon: true,
```

Comment:
`beacon: true` doesn't actually do anything.

Reply:
:) That's what happens when I copy/paste :)

I think from the Links page. I'll clean it up.